client/js/socket-events/commands.js

```diff
@@ -1,8 +1,12 @@
 const constants = require("../constants");
+import localCommands from "../commands/index";
 import socket from "../socket";
 
+const clientCommands = Object.keys(localCommands).map((cmd) => `/${cmd}`);
```

Comment:
This just reminds me how inconsistent JS is again 😂 the need to call a class method on and passing in an instance of the said class to get the desired effect `Object.keys(obj)` instead of `obj.keys()` like in any other language. Well tbf, it's wrong to call it class methods in the first place as classes don't technically exist in JS anyways. But welp ¯\_(ツ)_/¯